### ISSUE TITLE
`vscode-solidity` is not handling `evm_version` correctly

### ISSUE THREAD

**Issue by zhiqiangxu** [juanfranblanco/vscode-solidity#444]
Here's the repo that's trigering this issue:

https://github.com/zhiqiangxu/es-op-batchinbox

<img width="1587" alt="image" src="https://github.com/juanfranblanco/vscode-solidity/assets/1265027/58c4517a-fb75-49ee-b79f-217d63880d59">

This error is the as error I get from foundry if `evm_version` is not configured [here](https://github.com/zhiqiangxu/es-op-batchinbox/blob/ff5700fefe97a80c4f358758bb9f81dd4691fad0/foundry.toml#L10).

**Comment by juanfranblanco** [comment]
Added an option on settings now, yes importing from foundry.toml will be ideal

**Comment by juanfranblanco** [comment]
@zhiqiangxu it is out now

**Comment by zhiqiangxu** [comment]
@juanfranblanco It works great! Is there a plan to import from foundry.toml ?

**Comment by juanfranblanco** [comment]
Yes there is, as the settings import directly, hopefully I get some time to do it soon.
